Bandwidth.Standard/Http/Client/HttpClientConfiguration.cs

```diff
@@ -131,11 +131,11 @@ public class Builder
             private TimeSpan maximumRetryWaitTime = TimeSpan.FromSeconds(0);
             private IList<int> statusCodesToRetry = new List<int>
             {
-                408, 413, 429, 500, 502, 503, 504, 521, 522, 524, 408, 413, 429, 500, 502, 503, 504, 521, 522, 524,
+                408, 413, 429, 500, 502, 503, 504, 521, 522, 524, 408, 413, 429, 500, 502, 503, 504, 521, 522, 524
             }.ToImmutableList();
             private IList<HttpMethod> requestMethodsToRetry = new List<string>
             {
-                "GET", "PUT", "GET", "PUT",
+                "GET", "PUT", "GET", "PUT"
```

Comment:
Why are these duplicated? #justAPIMaticThings?

Reply:
I'm not sure why that's the case but... It's been like that and I'm scared to change it.